src/frequenz/channels/util/_timer.py

```diff
@@ -408,15 +409,29 @@ def __init__(
                 instance is created. This can only be `True` if there is
                 already a running loop or an explicit `loop` that is running
                 was passed.
+            start_delay: The delay before the timer should start. If `auto_start` is
+                `False`, an exception is raised. This has microseconds resolution,
+                anything smaller than a microsecond mean no delay.
```

Comment:
Optionally you might want to fix all occurrences of the following cosmetic suggestion
```suggestion
                anything smaller than a microsecond means no delay.
```